ssh.go

```diff
@@ -167,7 +172,13 @@ func (c *SSH) Connect() error {
 		config.Auth = append(config.Auth, ssh.PublicKeys(pubkeySigners...))
 	}
 
-	dst := fmt.Sprintf("%s:%d", c.Address, c.Port)
+	// add IPv6 Support
```

Comment:
Unnecessary and perhaps confusing comment, especially if we have the `isIPV6()` function in place.

It will be quite self-explanatory:

```go
	if c.isIPV6() {
		dst = fmt.Sprintf("[%s]:%d", c.Address, c.Port)
	} else {
		dst = fmt.Sprintf("%s:%d", c.Address, c.Port)
	}
```

Reply:
Thx for the advice.  I´m appreciate it xD I´m not particular good in Commenting right now xD